.github/CODEOWNERS

```diff
@@ -130,42 +124,50 @@
 /sdk/digitaltwins/ @sjiherzig @johngallardo @olivakar
 
 # PRLabel: %Event Grid
-# AzureSdkOwners: @sarangan12 
-# ServiceLabel: %Event Grid
 /sdk/eventgrid/ @sarangan12
 
+# AzureSdkOwners: @sarangan12
+# ServiceLabel: %Event Grid
```

Comment:
@jsquire I see `@Kishp01 @ahamad-MS` as service owners for Event Grid in .NET repo.  Do I use the same here?

Reply:
I'd go with `robece` who is the new EG PM